bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingImpl.java

```diff
@@ -43,6 +48,8 @@
 
     private Configuration configuration = new Configuration();
 
+    private Map<String, String> properties = new HashMap<>();
```

Comment:
I would not introduce a Properties domain object only because there is a Configuration domain object available. Which logic should be inside a Properties domain object apart from a Map facade?